### ISSUE TITLE
Cannot read properties of undefined (reading 'then') v6.4.2/tinymce.min.js (Salesforce + tinymce)

### ISSUE THREAD

**Issue by mrdiamod** [tinymce/tinymce-docs#2784]
v6.4.2/tinymce.min.js
(url - not possible - closed project)
What is the problem and why is it so?

I work in the Salesforce platform. It so happened that it was necessary to switch from the old version to the new one, v6.4.2, there are no such problems in the old version.
There are no errors when loading the component and opening it, etc. I have connected the "image" plugin when I try to upload an image.jpg or any other image, I get an error. What is the problem and why so, how to fix it?
Then I need to upload via a special connector to Salesforce and take the uploaded image from there.
At the same time, errors also do not pop up from the dev console itself.
![image](https://github.com/tinymce/tinymce-docs/assets/28936973/5944225f-9652-4e6e-be28-454bd299c9b7)

**Comment by HarroF** [comment]
@mrdiamod  I have the same issue.
Did you manage to resolve this? :)

**Comment by mrdiamod** [comment]
> @mrdiamod I have the same issue. Did you manage to resolve this? :)

Hi! Yes, I found a solution to the problem, it was a rather strange situation both on my part and on the part of the developers, every time in the search I was given an older version (5+) than I migrated from the old to the new existing one (6.4.2). And according to the documentation, everything was OK, but then I manually went to set up a new version and got to the "loading images" tab and as it turned out, the developers simplified the acceptance of parameters instead:
function uploadImageToDocument(param1, param2, param1, param3, param4 )

has become:
function uploadImageToDocument(param1, param2)

Each version has different parameters for reception and this is confusing.
![image](https://github.com/tinymce/tinymce-docs/assets/28936973/50c478e7-1972-4215-b267-6e97b9b5e3df)
